templates/base.html

```diff
@@ -129,6 +129,7 @@
   </section>
 
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.7.3/highlight.min.js"></script>
```

Comment:
The latest version of hightlight.js is 11.7.0 . Why we are chosen older version 10.7.3?

Refer https://cdnjs.com/libraries/highlight.js

Reply:
I [resolved this change request](https://github.com/arulrajnet/attila/pull/82/files).